showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseCategoriesScreen.kt

```diff
@@ -1,40 +1,49 @@
 package com.airbnb.android.showkase.ui
 
-import androidx.activity.ComponentActivity
+import androidx.appcompat.app.AppCompatActivity
 import androidx.compose.foundation.lazy.LazyColumnFor
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.MutableState
-import androidx.compose.ui.platform.LifecycleOwnerAmbient
+import androidx.compose.ui.platform.ContextAmbient
+import androidx.navigation.compose.navigate
+import androidx.navigation.NavHostController
 import com.airbnb.android.showkase.models.ShowkaseBrowserScreenMetadata
 import com.airbnb.android.showkase.models.ShowkaseCategory
 import com.airbnb.android.showkase.models.ShowkaseCurrentScreen
+import com.airbnb.android.showkase.models.clear
 import com.airbnb.android.showkase.models.clearActiveSearch
 import com.airbnb.android.showkase.models.update
 import java.util.Locale
 
 @Composable
 internal fun ShowkaseCategoriesScreen(
-    showkaseBrowserScreenMetadata: MutableState<ShowkaseBrowserScreenMetadata>
+    showkaseBrowserScreenMetadata: MutableState<ShowkaseBrowserScreenMetadata>,
+    navHostController: NavHostController
 ) {
-    val activity = (LifecycleOwnerAmbient.current as ComponentActivity)
-    
+    val activity = ContextAmbient.current as  AppCompatActivity
     LazyColumnFor(items = ShowkaseCategory.values().toList()) { category ->
         val defaultlLocale = Locale.getDefault()
         SimpleTextCard(
             text = category.name.toLowerCase(defaultlLocale).capitalize(defaultlLocale),
             onClick = {
                 showkaseBrowserScreenMetadata.update {
                     copy(
-                        currentScreen = when(category) {
-                            ShowkaseCategory.COMPONENTS -> ShowkaseCurrentScreen.COMPONENT_GROUPS
-                            ShowkaseCategory.COLORS -> ShowkaseCurrentScreen.COLOR_GROUPS
-                            ShowkaseCategory.TYPOGRAPHY -> ShowkaseCurrentScreen.TYPOGRAPHY_GROUPS
-                        },
                         currentGroup = null,
                         isSearchActive = false,
                         searchQuery = null
                     )
                 }
+                when(category) {
+                    ShowkaseCategory.COMPONENTS -> navHostController.navigate(
+                        ShowkaseCurrentScreen.COMPONENT_GROUPS.name
+                    )
+                    ShowkaseCategory.COLORS -> navHostController.navigate(
+                        ShowkaseCurrentScreen.COLOR_GROUPS.name
+                    )
+                    ShowkaseCategory.TYPOGRAPHY -> navHostController.navigate(
```

Comment:
I imagine you could make an extension function that takes the enum value as the param so you can do 
```
 navHostController.navigate(ShowkaseCurrentScreen.COLOR_GROUPS)
```